### PR TITLE
Mha2nnunet skip converted cases

### PR DESCRIPTION
**Pull request by joeranbosma**
I'm not sure if this works as expected still. Are errors/skipped cases logged?

### CODE REVIEW

File: src/picai_prep/mha2nnunet.py [hunk 1]
```diff
@@ -126,11 +150,14 @@ def process_and_write(self, scans_out_dir: Path, annotations_out_dir: Path):
...
             self.write_log(f'Wrote annotation to {destination_path}')
 
-    def compile_log(self):
+    def compile_log(self) -> Optional[str]:
         if self.settings.verbose == 0:
             # logging is disabled
             return None
 
+        if self.skip_conversion:
+            return f"Skipping {self.subject_id}, already converted."
```
Comment (snorthman) [inline src/picai_prep/mha2nnunet.py:159]: maybe only for settings.verbose == 1?

Comment (snorthman) [reply src/picai_prep/mha2nnunet.py:159]: I meant to write 2

Comment (joeranbosma) [reply src/picai_prep/mha2nnunet.py:159]: I think it's useful to log that a case is skipped, also to inform the user that we do not overwrite the images.

Comment (snorthman) [reply src/picai_prep/mha2nnunet.py:159]: I agree, but verbosity level 1 should only include failures. We even wrote it in the documentation stub:
```
- verbose: control logfile verbosity. 0 does not output a logfile,
                    1 logs cases which have critically failed, 2 logs all cases (may lead to
                    very large log files)
                    Default: 1
```

**Pull request merged**